### PR TITLE
Feature/authclient pages enhancement

### PR DESCRIPTION
**Pull request by franciscojho**
Installed tailwind and improved the design of the login and register client pages.

### CODE REVIEW

File: src/components/compound/Styles.module.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
-.pageLayout {
+/* .pageLayout {
```
Comment (Drleugim) [inline src/components/compound/Styles.module.css]: Remove commented code. If this is deprecated code. Uncommented but don't call it anywhere

File: src/pages/client/RegisterPage/RegisterPage.jsx [hunk 2]
```diff
@@ -1,98 +1,115 @@
...
+        <Layout className="flex flex-col md:flex-row">
+            <AuthHeader loginTheme={false}>
+                <Logo className="w-32 h-auto md:mb-2 md:fill-tertiary" />
+                <h1 className="text-2xl font-bold w-100 self-center hidden md:block">
+                    Encuentra personas de confianza para organizar tu hogar
+                </h1>
+            </AuthHeader>
+            <section className="flex flex-col h-full w-full md:min-w-[450px] md:overflow-auto">
+                <main className="flex justify-center h-full w-full items-baseline">
+                    <div className="flex flex-col justify-center w-full max-w-xl px-12 pt-8">
```
Comment (Drleugim) [inline src/pages/client/RegisterPage/RegisterPage.jsx:19]: I'd rather you not use `<div>` for different sections of your components. Not a necessary change, but please think about creating custom sections in the future

**Review by Drleugim — CHANGES_REQUESTED** [review]
Please review comments and make necessary changes

**Comment by franciscojho** [comment]
A minor fix is required prior pr

File: src/components/compound/Styles.module.css (hunk 1 above)
Comment (franciscojho) [reply src/components/compound/Styles.module.css]: Pushed a couple of commits to fix this and another issue presented while doing the PR.

**Review by Drleugim — APPROVED** [review]
LGTM

**Pull request merged**